spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/web/WebConfiguration.java

```diff
@@ -97,6 +101,11 @@ private String getH2Port(String url) {
 
 	@Override
 	public void onStartup(ServletContext servletContext) {
+		LongTaskTimer longTaskTimer = LongTaskTimer
+				.builder("spring.cloud.dataflow.server").description("DataFlow duration timer")
```

Comment:
Not sure if this description will be displayed anywhere. If it gets displayed, please change the name `DataFlow` to `Spring Cloud Data Flow`

Reply:
done